src/AppServices/DataSciencePage.tsx

```diff
@@ -1,55 +1,410 @@
+        {t("createClusterAction")}
+      </Button>,
+      <Button
+        onClick={() => setIsModalOpen(false)}
+        key="cancel"
+        variant="secondary"
+      >
+        {t("cancelClusterAction")}
+      </Button>,
+    ],
+  };
+
+  const getData = useCallback(async () => {
+    const { clusters, installableClusters } = await loadClusters();
+    if (installableClusters.length > 0) {
+      setClustersState({
+        mode: RHodsClusterAddonMode.Install,
+        clusters: installableClusters,
+      });
+    } else {
+      setClustersState({
+        mode:
+          clusters.length > 0
+            ? RHodsClusterAddonMode.Upgrade
+            : RHodsClusterAddonMode.Create,
+        clusters,
+      });
+    }
+  }, [loadClusters]);
+
+  const handleInstallModalOpen = () => {
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    getData();
+    setIsModalOpen(true);
+  };
+
+  const handleSelectCluster = (cluster: ClusterObject) => {
+    console.log(cluster);
+    setSelectedCluster(cluster);
+  };
 
-export const DataSciencePage: FunctionComponent = () => {
-  const { t } = useTranslation();
   return (
     <>
+      <Modal
+        isOpen={isModalOpen}
+        onClose={() => setIsModalOpen(false)}
+        variant="medium"
+        actions={modalActions[mode]}
+        title={t(clusterModalTitles[mode])}
+        description={
+          mode !== RHodsClusterAddonMode.Detecting ? (
+            <Title headingLevel="h2" size="md">
+              {t(clusterModalDescriptions[mode])}
+            </Title>
+          ) : undefined
+        }
+      >
+        <ClusterModalContent
+          upgradeStrategy={upgradeStrategy}
+          handleSelectCluster={handleSelectCluster}
+          selectedCluster={selectedCluster}
+          mode={mode}
+          clusters={clusters}
+          handleStrategyChange={setUpgradeStrategy}
+        />
+      </Modal>
       <MarketingPageHero
-        title={t("datascienceoverview:heroTitle")}
-        tagLine={t("datascienceoverview:heroTagline")}
-        description={t("datascienceoverview:heroDescription")}
+        title={t("heroTitle")}
+        tagLine={t("heroTagline")}
+        description={t("heroDescription")}
         cta={
           <Flex>
-            <Button
-              data-testid="hero-buttonTryIt"
-              ouiaId="button-rhods-tryit"
-              variant={ButtonVariant.secondary}
-              component="a"
-              target="_blank"
-            >
-              {t("datascienceoverview:heroTryItButton")}
+            <Button onClick={handleInstallModalOpen}>
+              {t("installButton")}
             </Button>
-            <Button
-              data-testid="hero-buttonLearnMore"
-              variant={ButtonVariant.link}
-              component="a"
-              href="https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-data-science"
+            <a
```

Comment:
We have a component to render external links [here](https://redhat-developer.github.io/app-services-ui-components/?path=/story/shared-externallink--external-link)